restapi/api/streaming_endpoints.go

```diff
@@ -1,15 +1,51 @@
 package api
 
 import (
-	"io"
-	"net/http"
-
 	"github.com/danielpaulus/go-ios/ios"
+	"github.com/danielpaulus/go-ios/ios/instruments"
 	"github.com/danielpaulus/go-ios/ios/syslog"
 	"github.com/gin-gonic/gin"
 	log "github.com/sirupsen/logrus"
+	"io"
+	"net/http"
 )
 
+// Notifications uses instruments to get application state change events. It will stream the events as json objects separated by line breaks until it errors out.
+// Listen                godoc
+// @Summary      uses instruments to get application state change events
+// @Description uses instruments to get application state change events
+// @Tags         general
+// @Produce      json
+// @Success      200  {object}  map[string]interface{}
+// @Router       /notifications [get]
+func Notifications(c *gin.Context) {
+	device := c.MustGet(IOS_KEY).(ios.DeviceEntry)
+	listenerFunc, closeFunc, err := instruments.ListenAppStateNotifications(device)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
please never log.Fatal ever in golang. 
If you look at what it does, it calls os.Exit(1), which means you will terminate the process immediately.

Reply:
The only place where you can do that is at the top level of a CLI command, when the program is about to exit anyway. But please never use this inside the modules. If somebody imports go-ios into their go application, we would end up killing their program :-D
Also never use it inside servers, as you would kill the entire REST API with it.

Reply:
Sorry. These changes are coming from an unintended merge. I reverted them. It's no longer in the PR.